target/linux/octeon/image/Makefile

```diff
@@ -29,6 +29,15 @@ define Device/generic
 endef
 TARGET_DEVICES += generic
 
+ITUSROUTER_CMDLINE:=console=ttyS0,115200 root=/dev/mmcblk1p2 rootfstype=squashfs,ext4,f2fs rootwait
+define Device/itus_shield-router
+  DEVICE_VENDOR := Itus Networks
+  DEVICE_MODEL := Shield Router
+  CMDLINE := $(ITUSROUTER_CMDLINE)
+  IMAGE/sysupgrade.tar/squashfs += | append-metadata
```

Comment:
@Grommish Is there a reason why you add metadata here?
Just got aware because SUPPORTED_DEVICES now has a global default, so this will actually add metadata now ...

Can you comment a little bit on this line?